.travis.yml

```diff
@@ -7,8 +7,10 @@ python:
 install:
   - pip install -r dev-requirements.txt
   - pip install -e .
+before_script:
+  - docker-compose up -d
 script:
-  - flake8 graphql_compiler/
+  - flake8 --exclude **/snap_*.py graphql_compiler/
```

Comment:
consider marking `snap_*.py` as excluded from coverage as well -- there's no point in tracking their coverage